lib/handler.js

```diff
@@ -66,6 +67,10 @@ module.exports = function(options) {
                 _id: id
               };
 
+              if (apiVersion >= 6 && !actionDescriptionObj._type) {
```

Comment:
According to the docs, `apiVersion` is a string and not just a major version. Thus, it makes sense to use `semver` to compare versions.